cumulus/client/consensus/common/Cargo.toml

```diff
@@ -17,7 +16,5 @@ sp-trie = { git = "https://github.com/paritytech/substrate", rev = "c364008a6c7d
 # Other deps
 async-trait = "0.1.52"
 dyn-clone = "1.0.5"
-futures = { version = "0.3.21", features = ["compat"] }
-tracing = "0.1.32"
```

Comment:
Should we add checks for unused crates in CI?

Reply:
We could if it was reliable. It has false-positives sometimes and lately just crashes for me. Not to say it takes A LOT of time to run, probably even longer than compilation from scratch on my machine.